spec/support/shared_examples_for_phone_validation.rb

```diff
@@ -1,3 +1,6 @@
+require 'shoulda/matchers'
+include Shoulda::Matchers::ActiveModel
```

Comment:
I think we need to move this inside the `shared_examples` block otherwise it will leak into the global namespace right?